instructure/canvas-android
# [MBL-15400][Student] Student Annotation submission type

Test plan:
- Verify that the student annotation submission type is working. If there are more submission type the user should see it as an option and selecting it will redirect to the submission screen. If it's the only submission type, the user should be redirected without showing the dialog.
- Verify that the student can view the already submitted student annotation submission, but cannot modify it.
- Verify that the student cannot modify teacher's annotations on a file upload submission.
- Test the DB schema migration. Install an older version of the app, then update the app with the build from this branch. Verify that the submissions db doesn't break.

refs: MBL-15400
affects: Student
release note: Added support for student annotation submission type
